src/app/api/media/[...file]/route.ts

```diff
@@ -0,0 +1,15 @@
+import MediaService from '@/services/mediaService';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function GET(
+  _req: NextRequest,
+  ctx: { params: { file: string[] } }
+) {
+  const res = await MediaService.load(ctx.params.file.join('/'));
+
+  const headers = new Headers();
+  headers.set('Content-Type', 'image/*');
```

Comment:
By adding jpeg or png to the end of the mime type. The image can be displayed directly in the browser.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#image